Documentation/HowToGuides/AddingACollisionIgnorer/README.md

```diff
@@ -0,0 +1,64 @@
+# Adding A Collision Ignorer
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The `Input Mutator ColliisonIgnorer` provides a way of allowing two or more objects to not interact with each other.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 0`, `Z = 0`
+
+![Adding A Cube](assets/images/AddingACube.png)
+
+### Step 2
+
+Create another new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 2`, `Z = 0`
+
+![Adding The Second Cube](assets/images/AddingTheSecondCube.png)
+
+### Step 3
+
+Click the `Add Component` button and select the `Rigidbody` component to the second `Cube`. 
+
+![Adding A Rigidbody](assets/images/AddingARigidbody.png)
+
+Play the `Unity Scene` and you should see the second `Cube` land on top of the first `Cube`.
+
+![Playing The Unity Scene With The Rigidbody](assets/images/PlayingTheUnitySceneWithTheRigidbody.png)
+
+### Step 4
+
+Expand the `Tilia Input Mutators CollisionIgnorer` Package directory in the Unity Project window and select then `Packages -> Tilia Input Mutators CollisionIgnorer -> Runtime -> Prefabs -> Actions` directory then drag and drop the `Input.Mutators.ColliisonIgnorer` prefab into the Hierarchy window.
+
+![Drag And Drop Collision Ignorer Into Hierachy](assets/images/DragAndDropCollisionIgnorerIntoHierachy.png)
+
+
+### Step 5
+
+Under `Sources` in the `Rigidbody` settings change the `size` value to `1` and drag and drop the first `Cube` into the box displaying the name `Element 0`.
```

Comment:
Just replace this copy with a pretty much straight up copy/paste from Step 4 of the Object Follower doc:

https://github.com/ExtendRealityLtd/Tilia.Mutators.ObjectFollower.Unity/blob/master/Documentation/HowToGuides/AddingAnObjectFollower/README.md#step-4



Select the `Mutators.CollisionIgnorer` GameObject in the Unity Hierarchy window then increase the `Sources -> Elements -> Size` property by `1` on the `Collision Ignorer` component.

> The size property will be `0` by default so change it to `1`.

Drag and drop the `Cube` GameObject from the Unity Hierarchy window into the newly displayed Element `0` field within the `Sources -> Elements` parameter on the `Collision Ignorer` component.